constraints.txt

```diff
@@ -25,21 +25,21 @@ persistent==5.0
 pycparser==2.21
 python-gettext==4.1
 pytz==2022.7.1
-roman==3.3
+roman==4.0
 six==1.16.0
 soupsieve==2.3.2.post1
 transaction==3.0.1
 waitress==2.1.2
 z3c.pt==3.3.1
 zExceptions==4.3
-zc.lockfile==2.0
+zc.lockfile==3.0.post1
```

Comment:
@icemac I notice you created that `post1` release. Please let's avoid silly version numbers like that.

Reply:
@dataflake I do not think this is a silly version number. According to https://peps.python.org/pep-0440/#post-releases these kind of version numbers can be used if the actual code did not change but there was a packaging problem. The change log of this package shows that `python_requires` was missing, which does not affect the package itself but allowed to install it on too old versions. (See https://pypi.org/project/zc.lockfile/#post1-2023-02-28.)